rs/nns/governance/src/governance.rs

```diff
@@ -5625,11 +5625,22 @@ impl Governance {
         }
     }
 
+    /// Preconditions:
+    /// 0. `register_vote.proposal` is a proposal ID of a proposal that still accepts votes.
+    /// 1. Neuron identified by `neuron_id` is authorized to vote on that proposal
+    ///    (either due to being the controller or a hotkey of a neuron that has a ballot).
+    /// 2. `register_vote.vote` must not be `Vote::Unspecified`.
+    /// 3. The neuron has not already cast its vote.
+    ///
+    /// Practically, neurons that have already dissolved cannto vote, so long as the minimal
+    /// possible dissolve delay is greated than the maximum possible voting period. Refer to:
+    /// - `VotingPowerEconomics.NEURON_MINIMUM_DISSOLVE_DELAY_TO_VOTE_SECONDS_BOUNDS`
```

Comment:
Should we have this "warning" next to where this parameter is defined (or rather where its floor is defined)? This is where I would have expected it to run over the comment if we ever change the boundaries of which value this can take on..

Reply:
Done